test/core/gear_log/file_handle_test.exs

```diff
@@ -46,6 +46,15 @@ defmodule AntikytheraCore.GearLog.FileHandleTest do
     assert_file_content([log_msg])
   end
 
+  test "write/2 should rescue and continue processing normally" <>
+    "when process crashed by receiving data that is not UTF8 string" do
```

Comment:
l.50とl.51の行頭の位置が同じなのは許せません。このレイアウトはやめましょう。
行が長くなっても構わないので単につなげればいいですが、
`"write/2 should skip malformed (non-UTF8) log message"`などと短くしてもOK。

Reply:
Sorry, it certainly wasn't beautiful.
fixed in fd43d4d

BTW, in case the explanation will unavoidably become long (for example):
```
test "write/2 should rescue and continue processing normally" <>
     "when process crashed by receiving data that is not UTF8 string" do
```
is OK?

Reply:
Personally I don't care if a line is a bit too long (code layout and multi-line alignment has precedence), so just put it into one line is OK as long as the test description is reasonable.
The code above is OK for me.
If a long test description really matters then we could also introduce a `describe` block.

Reply:
OK, thank you!